src/DssSpell.sol

```diff
@@ -14,30 +14,19 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity 0.6.11;
+import "dss-exec-lib/DssExec.sol";
```

Comment:
Was it in purpose that we do the imports before the pragma?

Reply:
I would put pragma first

Reply:
oversight, fixed